x/swaprouter/router_test.go

```diff
@@ -6,19 +6,22 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
 	gamm "github.com/osmosis-labs/osmosis/v13/x/gamm/keeper"
+	"github.com/osmosis-labs/osmosis/v13/x/gamm/pool-models/balancer"
 	poolincentivestypes "github.com/osmosis-labs/osmosis/v13/x/pool-incentives/types"
 	"github.com/osmosis-labs/osmosis/v13/x/swaprouter/types"
 	swaproutertypes "github.com/osmosis-labs/osmosis/v13/x/swaprouter/types"
 )
 
 const (
-	denomA = "denomA"
-	denomB = "denomB"
-	denomC = "denomC"
+	foo   = "foo"
+	bar   = "bar"
+	baz   = "baz"
+	uosmo = "uosmo"
```

Comment:
nit: noticed that there are places where hardcoded values are still used. Please make sure to use these variables everywhere where applicable